tests/test_mixed.py

```diff
@@ -0,0 +1,1952 @@
+import sys
```

Comment:
I haven't read these; should I?

One of the concerns I have is that when ppl implement new features, they will forget to deal w/ the subtree problem; I'm wondering if we could add a test here that takes *all* the features, and compares running it against a very simple homogenous and mixed tree, and checks if the results are different?  That way the test would fail if someone forgets (in theory)

Reply:
The tests look into the features dictionaries and raise if there are new tests that are not included.
```python
    features_not_tested = list(
        set(_NEURITE_FEATURES) - set(features.keys()) - set(_MORPHOLOGY_FEATURES)
    )

    assert not features_not_tested, (
        "The following morphology tests need to be included in the tests:\n\n" +
        "\n".join(sorted(features_not_tested)) + "\n"
    )
```